### DISCUSSION TITLE
Como criar um sistema de construção com html, css e tbm pode usar python ou js?

### DISCUSSION THREAD

**Discussion by rafgam10** [frontendbr/forum#2377] [Perguntas e Respostas]
Quero criar um sistama de construção por meu pai, pois ele reclamar muito o sistema que ele usar na loja, queria sabe como criar um sistema pelo navegador e deixar ligado direto, o que precisa necessariamente?

Msg: Se tiver algum vidéo que pode me ajudar prf mandar o link.

**Comment by guilhermesantoss** [comment]
A stask javascript resolve seu problema, React, Vue ou Angular para o frontend, se for fazer todo o desenvolvimento sozinho no BE recomendo o NodeJS, banco de dados é preciso fazer uma análise. Da uma olhada na questão de hospedagem tbm, domínio e etc, na infra do seu projeto.

Procura pelo conteúdo da Rocketseat que tem bastante coisa de React e NodeJS.
